beets/ui/commands.py

```diff
@@ -79,6 +79,26 @@ def _do_query(lib, query, album, also_items=True):
     return items, albums
 
 
+def _paths_from_logfile(path):
+    """Parse the logfile and yield skipped paths to pass to the `import`
+    command.
+    """
+    with open(path, mode="r", encoding="utf-8") as fp:
+        for line in fp:
+            verb, sep, paths = line.rstrip("\n").partition(" ")
+            if not sep:
+                raise ValueError("malformed line in logfile")
```

Comment:
I'm not sure `ValueError` is the right thing here. I believe we want to surface this kind of error to the user (assuming we change the interface to be explicit about the file type), so maybe `ui.UserError` is more appropriate—or a new subclass thereof?

Reply:
I thought that at this point the error is "internal". `os.path.commonpath` might raise a `ValueError` too. The error is caught in the calling code and could be used to raise a `ui.UserError`. I didn't do that yet because without an additional flag we can't  distinguish logfiles from audio files yet.

Reply:
error handling is here: https://github.com/beetbox/beets/pull/4387/commits/99ced5cc7def450181cc62fb7190be5ef4697eba#diff-e324b20657a7d6b43b8b7aeb5754b96774f5062294b5ba7f1e3062845e9e7044R982-R991